source/Dialog.cpp

```diff
@@ -129,18 +177,17 @@ void Dialog::Draw()
 	const Sprite *bottom = SpriteSet::Get(isWide ? "ui/dialog bottom wide" : "ui/dialog bottom");
 	const Sprite *cancel = SpriteSet::Get("ui/dialog cancel");
 
-	// Get the position of the top of this dialog, and of the text and input.
-	Point pos(0., (top->Height() + height * middle->Height() + bottom->Height()) * -.5f);
-	Point textPos(Width() * -.5 + 10., pos.Y() + 20.);
-	Point inputPos = Point(0., -70.) - pos;
+	// Get the position of the top of this dialog, and of the input.
+	Point pos(0., (top->Height() + extensionCount * middle->Height() + bottom->Height()) * -.5f);
```

Comment:
```suggestion
	Point pos(0., (top->Height() + extensionCount * middle->Height() + bottom->Height()) * -.5);
```
point takes in double, no idea why it was float here before